library/formatting.inc.php

```diff
@@ -216,3 +216,27 @@ function oeFormatAge($dobYMD, $nowYMD = '', $format = 0)
 
     return $age;
 }
+
+
+/**
+ * Returns the complete formatted stae string according the global date format
+ * @param $timestamp
+ * @return bool|string
+ */
+function oeTimestampFormatDateTime($timestamp)
+{
+    if(!$timestamp){
+        $timestamp = strtotime(date('Y-m-d H:i'));
+    }
+
+    if ($GLOBALS['date_display_format'] == 1) {      // mm/dd/yyyy, note year is added below
+        $newDate = date('m-d-Y H:i',$timestamp);
```

Comment:
in here to support the time locale setting, would do the following within each if:
``````
if ($GLOBALS['time_display_format'] == 0) {
    $newDate = date('m/d/Y H:i',$timestamp);
}  else { // $GLOBALS['time_display_format'] == 1
    $newDate = date('m/d/Y g:i a',$timestamp);
}